Source/WebCore/platform/ScrollableArea.h

```diff
@@ -424,6 +425,8 @@ class ScrollableArea : public CanMakeWeakPtr<ScrollableArea> {
     virtual void updateScrollAnchoringElement() { }
     virtual void updateScrollPositionForScrollAnchoringController() { }
     virtual void invalidateScrollAnchoringElement() { }
+    virtual void updateScrollbarsControllerForLayerCreation() { }
```

Comment:
I'm not sure what the "for layer creation" implies here.

Reply:
Changed this to be `updateScrollbarsControllerAfterLayerCreation`.